script/downsize_uploads.rb

```diff
@@ -77,23 +32,72 @@
     new_file = false
   end
 
+  before = upload.filesize
   upload.filesize = File.size(path)
+  delta = upload.filesize - before
```

Comment:
You don't actually need that `delta` variable ;)

```ruby
return if before < upload.filesize
```

or 

```ruby
return if upload.filesize > before
```

Reply:
Ah, yes, that was a leftover from a *log-everything* version 😉